spaces/domains.json

```
@@ -81,6 +81,7 @@
   "vote.acryptos.com": "acryptos",
   "vote.balancer.fi": "balancer.eth",
   "vote.balancer.finance": "balancer.eth",
+  "vote.meso.finance":"mesofinance.eth",
```

Please add a space between domain and ENS value.
```suggestion
  "vote.meso.finance": "mesofinance.eth",
```